R/Connect.R

```diff
@@ -110,7 +183,7 @@ getJbcDriverSingleton <- function(driverClass = "", classPath = "") {
 
 findPathToJar <- function(name, pathToDriver) {
   if (missing(pathToDriver) || is.null(pathToDriver)) {
-    pathToDriver <- system.file("java", package = "DatabaseConnectorJars")
+    pathToDriver <- jdbcDrivers$defaultPath
```

Comment:
Does `jdbcDrivers$defaultPath` have a value? I think this code is now obsolete?

Reply:
Yes. You're correct. I don't have a good way to run the automated tests for this package yet. I need to get that set up which will help me catch errors before making commits!

Reply:
I think this is fixed but I have not tested the code in this PR. Hopefully I can get that set up soon. I might need to modify some of the tests by breaking them by database. (sql server, postgres, oracle, etc,  would be in separate tests so I can just run the tests on the databases I have access to.)